internal/indexer/indexer.go

```diff
@@ -127,8 +148,54 @@ func (i *Indexer) Index() error {
 	return nil
 }
 
+func (i *Indexer) startImportMonikerReferenceTracker(wg *sync.WaitGroup) {
+	go func() {
+		contained := struct{}{}
+
+		wg.Add(1)
```

Comment:
`wg.Add(1)` must be called _outside_ of the goroutine that you are protecting, otherwise the scheduler can see a zero count before the goroutine is scheduled and exit.